index.Rmd

```diff
@@ -153,126 +157,121 @@ the assignment symbol is preferred and is what we will use in this course. When
 it with quotes like the filename. This is because we are formally telling R about this word and giving it a value. Only characters and words that act as values need 
 to be surrounded by quotes.
 
-Let's now use the assignment symbol to give the name `us_data` to the US state-level property, income, population, and voting data frame that we get from `read_csv`.
+Let's now use the assignment symbol to give the name `can_lang` to the language data collected in the 2016 Canadian census data frame that we get from `read_csv`.
 
 ```{r load_data_with_name, message=FALSE}
-us_data <- read_csv("data/state_property_vote.csv")
+can_lang <- read_csv("data/can_lang.csv")
 ```
 
 Wait a minute! Nothing happened this time! Or at least it looks like that. But actually something did happen: the data was read in and now has the name `us_data` associated with it. 
 And we can use that name to access the data frame and do things with it. First we will type the name of the data frame to print it to the screen.
 
 
 ```{r print}
-us_data
+can_lang
 ```
 
 ## Creating subsets of data frames with `select` & `filter`
 
 Now, we are going to learn how to obtain subsets of data from a data frame in R using two other `tidyverse` functions: `select` and `filter`. 
 The `select` function allows you to create a subset of the columns of a data frame, while the `filter` function allows you to obtain a subset of the rows with specific values.
 
-Before we start using `select` and `filter`, let's take a look at the US state-level property, income, and population data again to familiarize ourselves with it. 
+Before we start using `select` and `filter`, let's take a look at the language data collected in the 2016 Canadian census again to familiarize ourselves with it. 
 We will do this by printing the data we loaded earlier in the chapter to the screen. 
 
 ```{r print_data_again}
-us_data
+can_lang
 ```
 
 
-In this data frame there are 52 rows (corresponding to the 50 US states, the District of Columbia and the US territory, Puerto Rico)
+In this data frame there are 214 rows rows (corresponding to the 214 languages recorded on the 2016 Canadian census)
 and 6 columns: 
 
-1. US state name
-2. Population
-3. Median property value
-4. Median household income
-5. Average commute time in minutes
-6. The party each state voted for in the 2016 US presidential election
-
+1. `category`: Higher level language category (describing whether the language is an Official Canadian language, an Aboriginal language, or a Non-Official and Non-Aboriginal language).
+2. `language`: Language queried about on the Canadian Census.
+3. `mother_tongue`: Total Count of Canadians from the Census who reported the language as their mother tongue. Mother tongue is generally defined as the language someone was exposed to since birth.
```

Comment:
```suggestion
3. `mother_tongue`: Total count of Canadians from the Census who reported the language as their mother tongue. Mother tongue is generally defined as the language someone was exposed to since birth.
```